### ISSUE TITLE
ui-routing.module.ts

### ISSUE THREAD

**Issue by VZhyrytskiy** [MaksimLioshynepam/Angular2-#7]
По содержанию этот файл очень похож на app-routing.module.ts.
Может есть какие-то отличия, которые я не заметил?

**Comment by MaksimLioshyn** [comment]
Нет, вы абсолютно правы :) Поскольку у меня нет практики в ангуляре на проекте, то пытаюсь найти как можно больше решений и попробовать их руками, что бы лучше отложилось в памяти, по этому и появляются полнейшие дубли некоторых файлов (как в данном случае) и просто забывают удалить их. Работает и хорошо
